marketplace.xml

```diff
@@ -13057,4 +13057,37 @@ Uses the MarkLogic Data Movement Java SDK. Works with MarkLogic 9.0 and above on
     </version>
   </versions>
 </market_entry>
+<market_entry>
+    <id>generatechart</id>
+    <type>Step</type>
+    <name>Generate charts</name>
+    <category>
+      <name>Enhancements</name>
+      <parent>
+        <name>PDI Plugins</name>
+      </parent>
+    </category>
+    <description>This plugin allow to generate charts with javascript libraries like Google chart.This plugin has been developed as a final project of computer engineerin at the university of Alicante,Spain.
+    </description>
+    <author>Daniel Pascual Parrondo</author>
+    <documentation_url>https://github.com/danielpascual95/GenerateChart-wiki/blob/master/wiki</documentation_url>
+    <versions>
+      <version>
+        <branch>Stable</branch>
+        <version>1.0</version>
+        <package_url>https://sourceforge.net/projects/stepcharts/files/StepCharts.rar/download?use_mirror=master&r=&use_mirror=master</package_url>
```

Comment:
the & character must be replaced with `&amp;`. Thanks

Reply:
updated. Thanks